resources/css/app.css

```diff
@@ -0,0 +1,61 @@
+h1 {
+    text-align: center;
+}
+h2 {
+    text-align: left;
+}
+div.container {
+    text-align: center;
+}
+ul.myUL {
+    display: inline-block;
+    text-align: left;
+}
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+li {
+    display: inline;
+}
+li {
+    float: left;
+}
```

Comment:
Try to be a bit more consistent in the use of empty lines. It is good practice to add an empty line between blocks, but you can do whatever you (or your team) likes. But, whatever you choose, do it consistently.